src/components/missions/Mission.js

```diff
@@ -50,7 +57,7 @@ Mission.propTypes = {
     singleMission: PropTypes.string,
     description: PropTypes.string,
     mission_id: PropTypes.string,
-
+    joined: PropTypes.string,
```

Comment:
- [ ] The prop `joined` has data that is boolean. The prop-type should  `PropTypes.bool` instead of `PropTypes.string`